File: app/src/main/java/dev/ragnarok/fenrir/model/SaveAccount.java
package dev.ragnarok.fenrir.model;

import androidx.annotation.Keep;

import com.google.gson.annotations.SerializedName;

@Keep
public class SaveAccount {
    @SerializedName("login")
    public String login;
    @SerializedName("password")
    public String password;
    @SerializedName("two_factor_auth")
    public String two_factor_auth;

    public SaveAccount set(String login, String password, String two_factor_auth) {
        this.login = login;
        this.password = password;
        this.two_factor_auth = two_factor_auth;
        return this;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/model/SaveAccount.kt
package dev.ragnarok.fenrir.model

import androidx.annotation.Keep
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Keep
@Serializable
class SaveAccount {
    @SerialName("login")
    var login: String? = null

    @SerialName("password")
    var password: String? = null

    @SerialName("two_factor_auth")
    var two_factor_auth: String? = null
    operator fun set(login: String?, password: String?, two_factor_auth: String?): SaveAccount {
        this.login = login
        this.password = password
        this.two_factor_auth = two_factor_auth
        return this
    }
}